.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+_data/ @TomerFi @bennypowers
+_includes/ @bennypowers
+assets/ @bennypowers
+feed/ @bennypowers
+/*.* @bennypowers @TomerFi
```

Comment:
you'll get a ping for all the dot-files and such

Reply:
only for ones on the root (including root folders).
these are mainly core files no?